vendor/vendor.json

```diff
@@ -467,12 +465,16 @@
 			"versionExact": "v10.15.4"
 		},
 		{
-			"checksumSHA1": "JVAdK/V4SFCsCdpRBd4hTq/Z3h4=",
+			"checksumSHA1": "b2lrPJRxf+MEfmMafN40wepi5WM=",
+			"path": "github.com/Azure/go-autorest/logger",
+			"revision": "a35eae345f69bbfbe3b8fa0b1d3fe98f8430b21a",
+			"revisionTime": "2018-08-30T19:44:05Z"
+		},
+		{
+			"checksumSHA1": "scpSozMdk4sqSpkbQqupLKUfLiM=",
 			"path": "github.com/Azure/go-autorest/version",
```

Comment:
Do we need to change it?

Reply:
I was unable to successfully run `make vendor-status` without this change.